### ISSUE TITLE
looking for info about Moonbase

### ISSUE THREAD

**Issue by Pyroseza** [Fenrir-23/Moon-Base-clone#1]
hey, do you remember the exact mechanics about how the game worked?

I'm considering making a Discord bot in Python so people can play across servers

But it's been so long and I cannot remember the exact details as to how it worked so I'm just trying ask whoever I can :)

**Comment by Albrecht-Ohsiek** [comment]
Hi Pyroseza, I don't remember much since its been more than a decade but I remember some, and I found a few sources online. I'm still trying to find out more information as well. 
![Moonbase](https://github.com/Albrecht-Ohsiek/Moon-Base-clone/assets/84184217/d05aecb6-88d1-4a8f-b20b-9ff89388cdbc)
Also see MXit API Launch Slideshow
https://www.slideshare.net/blueleafgames/mxit-api-launch-blue-leaf-games-5614183

**Comment by Pyroseza** [comment]
Yeah same, I remember many fond memories of staying up late and plotting attacks with friends, I wish I remembered more details. Thanks for sharing I’ll take a look 👀
